### PR TITLE
3926 public advanced order search ui

### PR DESCRIPTION
**Pull request by rachelschneiderman**
image

### CODE REVIEW

File: web-api/src/public-api/orderPublicSearchLambda.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const { genericHandler } = require('../genericHandler');
+
+/**
+ * used for fetching cases matching the given name, country, state, and/or year filed range for the general public
```
Comment (kkoskelin) [inline web-api/src/public-api/orderPublicSearchLambda.js]: for fetching *orders*

File: web-client/integration-tests-public/journey/unauthedUserSearchesForOrderByKeyword.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+export default (test, testClient) => {
+  return it('Search for order by keyword', async () => {
+    test.setState('advancedSearchForm', {
+      orderSearch: {
+        orderKeyword: 'osteodontolignikeratic',
```
Comment (kkoskelin) [inline web-client/integration-tests-public/journey/unauthedUserSearchesForOrderByKeyword.js:5]: note: this word is unknown by the googles. 💯

**Review by kkoskelin — APPROVED** [review]

File: web-client/integration-tests-public/journey/unauthedUserSearchesForOrderByKeyword.js (hunk 2 above)
Comment (sutt0n) [reply web-client/integration-tests-public/journey/unauthedUserSearchesForOrderByKeyword.js:5]: however, `osteodontokeratic` is 😄

**Review by sutt0n — APPROVED** [review]

**Pull request merged**